flexmeasures/api/v3_0/tests/test_sensors_api.py

```diff
@@ -44,7 +44,7 @@
             None,
             None,
             "power",
-            2,
+            3,
```

Comment:
Good find! It's of course unintentional that we have this kind of interrelationships between what should have been independent tests, but maybe to help future devs we could at least add a helpful message at the assert that fails when adding a new sensor. So I suggest revising line 199 to `assert response.json["num-records"] == exp_num_results, f"If this line fails, a conftest may have added another sensor accessible to {requesting_user}. Update the exp_num_results in the test parameters accordingly."`